Makefile

```diff
@@ -163,7 +163,6 @@ vet: ## Run go vet against code.
 ENVTEST := $(shell pwd)/bin/setup-envtest
 ENVTESTPATH = $(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(shell pwd)/bin -p path)
 ifeq ($(shell $(ENVTEST) list | grep $(ENVTEST_K8S_VERSION)),)
-# TODO what --arch=amd64 does here?
 	ENVTESTPATH = $(shell $(ENVTEST) --arch=amd64 use $(ENVTEST_K8S_VERSION) -p path)
```

Comment:
In some past version there was an envtest_k8s_version value that would only support amd64 since there's no arm64 artifacts available to download.